wildfly-cloud-legacy-galleon-pack/examples/jaxws/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wildfly.demos</groupId>
+    <artifactId>jaxws-pojo-service</artifactId>
+    <version>1.0.0.Final-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>WildFly Bootable JAR - JAX-WS</name>
+    <description>This project is a working example of a JAX-WS web service endpoint; this is the service POM file</description>
```

Comment:
Just asking, wouldn't be better a JAX-RS example instead? A JAX-RS is easy to test by using the browser / curl

Reply:
The example is just a POC one. It uses currently Bootable JAR Maven plugin, I am using it as a quick way to test the feature-pack manually. The actual examples would be different.